packages/inter-protocol/test/vaultFactory/vaultFactoryUtils.js

```diff
@@ -58,7 +58,9 @@ export const defaultParamValues = debtBrand =>
  * @param {Array<NatValue> | Ratio} priceOrList
  * @param {RelativeTime} quoteInterval
  * @param {Amount | undefined} unitAmountIn
- * @param {bigint} [startFrequency]
+ * @param {object} actionParams
+ * @param {bigint} actionParams.startFrequency
+ * @param {bigint} actionParams.discountStep
```

Comment:
+1 to using the params object. but please use the `AuctionParams` type.
```suggestion
 * @param {Pick<AuctionParams, 'StartFrequency' | 'DiscountStep'>} actionParams
```
Note that test-vaultFactory depends on this function signature too

Reply:
thanks for catching that. done.